governance.md

```diff
@@ -0,0 +1,393 @@
+
+When a subcommittee is proposed, it must include an intended scope over which
+that subcommittee will focus. Within that scope the subcommittee is trusted to
+make decisions on behalf of the larger group (through the standard voting
+procedure). Though they are encouraged to check-in with the steering council to
+provide clarity on direction. For items outside of the subcommittee's scope,
+they must defer to the steering council and/or appropriate subcommittee(s) (if
+they exists).
+
+Like the Council as a whole, subcommittees should conduct their business in an
+open and public manner unless privacy is specifically called for. Private
+subcommittee communications should happen on the main private mailing list of
+the Council unless specifically called for.
+
+The BDFL retains override authority over subcommittees, but should typically
+appoint a delegate who plays that role most of the time. This delegate must
+also be a Steering Council Member.
+
+### NumFOCUS Subcommittee
+
+The Council will maintain one narrowly focused subcommittee to manage its
+interactions with NumFOCUS.
+
+-   The NumFOCUS Subcommittee is comprised of 5 persons who manage project
+    funding that comes through NumFOCUS. It is expected that these funds will
+    be spent in a manner that is consistent with the non-profit mission of
+    NumFOCUS and the direction of the Project as determined by the full
+    Council.
+-   This Subcommittee shall NOT make decisions about the direction, scope or
+    technical direction of the Project.
+-   This Subcommittee will have 5 members. No more than 2 Subcommitee Members
+    can report to one person through employment or contracting work (including
+    the reportee, i.e. the reportee + 1 is the max). This avoids effective
+    majorities resting on one person.
+
+
+Partners and Funding
+====================
+
+Institutional Partners
+----------------------
+
+The Steering Council and BDFL are the primary leadership for the project. No
+outside institution, individual or legal entity has the ability to own,
+control, usurp or influence the project other than by participating in the
+Project as Contributors and Council Members.  However, it is important to
+recognize Institutional Partners which are critical in funding, disseminating,
+and broadly applying the project to have a positive social impact.
+
+An Institutional Partner is any recognized legal entity that consistently and
+meaningfully supports the development and maintenance of the Project.  This
+support may be in the form of employing Project Contributors, providing
+critical services like community training or infrastructure, or providing
+direct financial support to the Project.  This support must be for the general
+development and maintenance of the Project, and benefit a wide variety of users
+of the Project.  This support must be substantial and sustained, accounting for
+the employment of the equivalent of one maintainer, over several months.
```

Comment:
"one fulltime maintainer"?  Probably should specify the number of months.  6 months?

Reply:
It's a good point. Why choose 6 months? Also wouldn't 6 months and full-time maintainer conflict?

Reply:
I think it's fine to keep things vague and at the discretion of the council.

Also, I don't want to open a new conversation here.  It's time to wrap this up.

Reply:
SGTM. @mmccarty, if this is a concern that you would like to follow-up on, could you please raise a new issue on this repo? If there is no need to follow-up on this further, let's go ahead and mark this as resolved. Thanks both for your engagement here.

Reply:
These are just things that I thought of while reading.  I agree with @mrocklin that its fine to keep it vague here.

Reply:
Cool, thanks for checking in. Will go ahead and mark this as resolved then.